portfolio/src/main/webapp/index.html

```diff
@@ -1,42 +1,52 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" />
     <title>Bless Paynes</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
   </head>
   <body>
     <div class="center" id="content">
-    <div class="button" id="buttons">
+      <div class="button" id="buttons">
         <p class="center font" id="welcome">Bless Paynes' Portfolio</p>
         <a href="#About" class="button" id="about">About Me</a>
```

Comment:
I see you're using the `id="about"` in 2 places (here and on a div below). IDs must be unique, so what do you think of converting it to a `class` instead?